lib/dotcom_web/components/trip_planner/itinerary_detail.ex

```diff
@@ -63,35 +64,26 @@ defmodule DotcomWeb.Components.TripPlanner.ItineraryDetail do
 
   defp specific_itinerary_detail(assigns) do
     assigns =
-      assign(
-        assigns,
-        :all_routes,
-        assigns.itinerary.legs
-        |> Enum.reject(&match?(%PersonalDetail{}, &1.mode))
-        |> Enum.map(& &1.mode.route)
-      )
+      assigns
+      |> assign(:start_place, List.first(assigns.itinerary.legs).from)
+      |> assign(:start_time, List.first(assigns.itinerary.legs).start)
+      |> assign(:end_place, List.last(assigns.itinerary.legs).to)
+      |> assign(:end_time, List.last(assigns.itinerary.legs).stop)
```

Comment:
This whole blob is kind of gross, but @thecristen and I talked offline, and we're going to remove it in a fast-follow PR, so that this one doesn't get bloated.

Reply:
Side note, at least the variable name lengths decrement by one letter per line, so they form a nice shape.